inserts.sql

```diff
@@ -0,0 +1,3398 @@
+    ('Fernando', 'Tatis Jr.', 'SS', 'R', 'R', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'SD')),
+    ('Trey', 'Wingenter', 'RP', 'R', 'R', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'SD')),
+    ('Matt', 'Wisler', 'SP', 'R', 'R', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'SD')),
+    ('Kirby', 'Yates', 'RP', 'R', 'L', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'SD'));
+
+INSERT INTO games
+    (startTime, homeScore, awayScore, homeTeamId, awayTeamId)
+
+VALUES
+    ('2019-06-06 12:35:00', '6', '1', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'PIT'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'ATL')),
+    ('2019-06-06 20:05:00', '4', '3', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'TEX'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'BAL')),
+    ('2019-06-06 13:15:00', '5', '7', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'KC'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'BOS')),
+    ('2019-06-06 14:20:00', '1', '3', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'CHC'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'COL')),
+    ('2019-06-06 13:15:00', '3', '1', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'STL'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'CIN')),
+    ('2019-06-06 19:00:00', '4', '5', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'CLE'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'MIN')),
+    ('2019-06-06 13:10:00', '1', '6', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'DET'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'TB')),
+    ('2019-06-06 15:40:00', '7', '8', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'SEA'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'HOU')),
+    ('2019-06-06 22:07:00', '4', '7', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'LAA'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'OAK')),
+    ('2019-06-06 14:10:00', '5', '1', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'MIL'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'MIA')),
+    ('2019-06-06 12:10:00', '7', '3', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'NYM'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'SF')),
+    ('2019-06-06 19:00:00', '2', '6', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'TOR'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'NYY')),
+    ('2019-06-06 22:10:00', '5', '4', (SELECT id
+        FROM teams
+        WHERE abbreviation = 'SD'), (SELECT id
+        FROM team
+        WHERE abbreviation = 'WSH'));
+
+
+INSERT INTO hitterStats
+    (atBats, runs, hits, doubles, triples, homeRuns, runsBattedIn, walks, strikeOuts, steals, playerId, gameId)
+
+VALUES
```

Comment:
Great job attempting the subqueries. The gameId one will work perfectly. You missed the playerId one though.